debug_tool/utils/db.py

```
@@ -98,23 +98,123 @@ def get_overview(self) -> Dict[str, Any]:
+            conn.close()
+            return {"count": int(row["cnt"]) if row else 0}
+        except Exception as e:
+            logger.error("[调试工具] SQLite集合统计查询失败: %s", e, exc_info=True)
+            return {"count": 0}
+
     def get_collections(self) -> List[str]:
         if not self.client:
             return []
         try:
             return [c.name for c in self.client.list_collections()]
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)